03-seeking-help.Rmd

```diff
@@ -124,6 +124,11 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find out which function and with what parameters should you use
+> to load data from csv file with columns delimited with "tab" and decimal point ".".
```

Comment:
Change: "from csv file" -> "from a csv file"

> columns delimited with "tab" and decimal point "."

Do you mean columns delimited with tab and a decimal point `\t.` or `\t` OR `.`? Perhaps reword to clarify this?